### DISCUSSION TITLE
Open Link in new tab

### DISCUSSION THREAD

**Discussion by alokVerma749** [mdx-js/mdx#2239] [Q&A] [answered]
want to open this link in new tab but got an error
![image](https://user-images.githubusercontent.com/87599400/215768889-a94a1842-f3be-4d36-9fd0-b3e191a1d61a.png)

ERROR:
![image](https://user-images.githubusercontent.com/87599400/215769050-c19b6607-c962-4c65-8278-d60c30621a10.png)

**Comment by wooorm** [comment, marked as answer]
That’s not markdown. It doesn’t work here on GH. It doesn‘t work in CommonMark. See [“What is MDX?”](https://mdxjs.com/docs/what-is-mdx/).

Also:

- You probably shouldn’t use `target=_blank`, you are breaking user expectation, about what clicking a link does.
- Don’t set *only* target blank: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target. That’s a security vulnerability
- Use JSX if you want to add attributes: `<a href="whatever" target="_blank" rel="noopener">...</a>`
- Or use a plugin: https://github.com/rehypejs/rehype-external-links.

**Reply by alokVerma749** [reply]
is there any way which can open link in a new tab

**Reply by ChristianMurphy** [reply]
Yes, by reading the response you commented on.
Particularly the last two bullet points, and trying the solutions.

**Reply by ascpixi** [reply]
How is using `target=_blank` "breaking user expectation, about what clicking a link does"? Sometimes you want to link to a document that the user is intended to read and then come back to the linking page (e.g. "install _(some program from this external website)_"). In this case, the logical `target` *is* indeed `_blank`. Where is this assumption coming from?

**Reply by remcohaszing** [reply]
Lift click uaually means to open in the current window. Middle click usually means to open in a new tab. Users have control and expectations how they want to open a page. By adding `target=_blank`, you take this away.

But that’s besides the point. The Commonmark spec doesn’t specify we should it. You can customize links with rehype plugins or custom components.
